src/directory/directory.js

```diff
@@ -93,11 +93,6 @@ const directory = {
             route: '/lib/analytics/enable-disable',
             filters: ['android', 'flutter', 'ios', 'js', 'react-native']
           },
-          {
-            title: 'Identify user',
-            route: '/lib/analytics/identifyuser',
-            filters: ['android', 'flutter', 'ios', 'react-native', 'js']
```

Comment:
This change seems unrelated to the rest, was it intended to remove android/flutter/ios as well? Those seem to have information

Reply:
Good catch Bannon.  This was a change tied to some of the Analytics changes I have locally that got included with the directory.js file updates.  While this is a needed removal, it isn't within scope of this PR overall.  I'll add it back in and save it to be included with all the Analytics changes.